eng/versioning/version_client.txt

```diff
@@ -176,6 +176,7 @@ com.microsoft:microsoft-opentelemetry-exporter-azuremonitor;1.0.0-beta.1;1.0.0-b
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
+unreleased_com.azure:azure-core;1.11.0-beta.1
```

Comment:
This looks good. Just making sure,  we will be going in this order...
GA - azure-core
GA - azure-core-amqp
GA - service-bus

Reply:
@hemanttanwar yes, that is the order for releasing. However, this PR will be merged only after SB is fully ready to GA.